types/rdfjs__formats/index.d.ts

```diff
@@ -1,12 +1,9 @@
-import { SinkMap } from "@rdfjs/sink-map";
-import { Stream } from "@rdfjs/types";
-import { EventEmitter } from "events";
 import Formats from "./lib/Formats.js";
 
-export { default as Formats } from "./lib/Formats.js";
+export { Formats } from "./lib/Formats.js";
```

Comment:
I made a mistake here before. This now matches the interface exported from `@rdfjs/formats-common` https://github.com/DefinitelyTyped/DefinitelyTyped/blob/a7ab291f7df846c2742ef6f52557a3da7ac1635b/types/rdfjs__formats-common/index.d.ts#L8-L11